src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -219,9 +229,20 @@ export default function MissingFileOverview({
   const [isChecked, setIsChecked] = useState(
     data?.consentToMailMissingRequiredFiles || false,
   );
+  useEffect(() => {
+    // Hides non-removable `&nbsp` label from VaCheckboxGroup on missing docs consent page
+    addStyleToShadowDomOnPages(
+      ['/consent-mail'],
+      ['va-checkbox-group'],
+      '.input-wrap .usa-fieldset .usa-legend {display: none}',
+    );
+  });
```

Comment:
Not able to remove the `&nbsp` ([added here](https://github.com/department-of-veterans-affairs/component-library/blob/6f952bdc1f4c39b77374fadded39d827aeb3fcbd/packages/web-components/src/components/va-checkbox-group/va-checkbox-group.tsx#L178)), so adding style to hide the element just on this page.

Reply:
Why doesn't this checkbox group need a `legend`? Is there a defect with the web component that is forcing you to do this workaround?

Reply:
I also would like to suggest using `va-checkbox` if it's just a single checkbox that you need: https://design.va.gov/storybook/?path=/docs/uswds-va-checkbox--docs#tile

Reply:
The checkbox group just has this single checkbox in it and the design didn't dictate a legend, so the `&nbsp` throws in extra spacing. I can definitely look into another workaround, I just had this utility function lying around so it was convenient to use it.

![Screenshot 2024-08-13 at 09 53 13](https://github.com/user-attachments/assets/c31e45f3-636b-46bb-b681-a3dcbef4d705)

Reply:
Looking at it again, seems I should just be using `VaCheckbox` not `VaCheckboxGroup`. I'll make that adjustment and take out the workaround. Thanks!

Reply:
Somehow I didn't see your second comment. Re-implemented with a single checkbox and ready for re-review. Thank you!

Reply:
Thanks for looking into it @michaelclement! Just for general awareness, there are accessibility reasons that the `legend` is required when using the `va-checkbox-group` component so it shouldn't ever really be necessary to try and hide that `&nbsp` as far as I know because the label/legend text should be there with it.

Of course if there are any doubts or something that seems like a defect in the design system, hit the team up in the #platform-design-system Slack channel. 🙌🏼

Reply:
Got it, that makes sense here. Thanks!